WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift

```diff
@@ -132,6 +150,29 @@ private extension LoginEpilogueViewController {
             topLine.isHidden = true
         }
     }
+
+    func setTableViewMargins() {
+        guard traitCollection.horizontalSizeClass == .regular &&
+            traitCollection.verticalSizeClass == .regular else {
+                tableViewLeadingConstraint.constant = defaultTableViewMargin
+                tableViewTrailingConstraint.constant = defaultTableViewMargin
+                return
+        }
+
+        let marginMultiplier = UIDevice.current.orientation.isLandscape ?
+            TableViewMarginMultipliers.ipadLandscape :
+            TableViewMarginMultipliers.ipadPortrait
+
+        let margin = view.frame.width * marginMultiplier
```

Comment:
The view hasn't been updated when this function is called, so the rotation was reported but the view's frame will have the old orientation's numbers.